File: Python/String-1/without_end.py

# given a string, return a version without the first and last char
# the string length will be at least 2
def without_end(str):
    s = ''

    if len(str) > 1:
        s = str[1:-1]

    return s

print(without_end('Hello'))
print(without_end('java'))
print(without_end('coding'))

File: JavaScript/String-1/withoutEnd.js

// given a string, return a version without the first and last char
// so 'Hello' yields 'ell'
// the string length will be at least 2
const withoutEnd = (str) => {
    return str.substring(1, str.length - 1);
};

console.log(withoutEnd('Hello'));
console.log(withoutEnd('java'));
console.log(withoutEnd('coding'));
